atests/test_ssl_certs.robot

```diff
@@ -5,19 +5,19 @@ Library     RequestsLibrary
 *** Test Cases ***
 Get HTTPS & Verify Cert
     [Tags]    get    get-cert
-    Create Session    httpbin    https://httpbin.org    verify=True
-    ${resp}=    GET On Session    httpbin    /get
+    Create Session    sslsession    https://github.com    verify=True
+    ${resp}=    GET On Session    sslsession    /
     Should Be Equal As Strings    ${resp.status_code}    200
 
 Get HTTPS & Verify Cert with a CA bundle
     [Tags]    get    get-cert
-    Create Session    httpbin    https://httpbin.org    verify=${CURDIR}${/}cacert.pem
-    ${resp}=    GET On Session    httpbin    /get
+    Create Session    sslsession    https://github.com    verify=${CURDIR}${/}cacert.pem
+    ${resp}=    GET On Session    sslsession    /
     Should Be Equal As Strings    ${resp.status_code}    200
 
 Get HTTPS with Client Side Certificates
     [Tags]    get    get-cert
     @{client_certs}=    Create List    ${CURDIR}${/}clientcert.pem    ${CURDIR}${/}clientkey.pem
-    Create Client Cert Session    crtsession    https://server.cryptomix.com/secure    client_certs=@{client_certs}
-    ${resp}=    GET On Session    crtsession    /
+    Create Client Cert Session    sslsession    https://github.com    client_certs=@{client_certs}
```

Comment:
I'm not sure about this. I thought that the server.cryptomix.com allowed a real test about ssl with a client certificate I'm wondering how can it work with github.com but I didn't write this original test so I'm unsure how it works,

Reply:
reverted the change for this test..

Reply:
Yep I was thinking the same. So did you find out anything?

Reply:
> Well, it looks like the client certs are not actually used in the … on session keywords. Will raise a new issue

Scrap this.. will keep looking. But would hope we can merge this one to also fix the pipeline of my other PR